src/manifest.json

```diff
@@ -14,6 +14,7 @@
             "*://www.google.co.uk/maps/*",
             "*://www.google.ru/maps/*",
             "*://www.google.com.ua/maps/*",
+            "*://www.google.cz/maps/*",
```

Comment:
There is also issue https://github.com/dudarev/mapjumper/issues/29 planned for milestone 0.7 (we are currently wrapping up 0.6) where we will add other Google TLDs as well.